README.md

```diff
@@ -6,19 +6,11 @@ IHT performs [feature selection](https://en.wikipedia.org/wiki/Feature_selection
 ## Installation
 
 IHT.jl is not registered in `METADATA`.
-It depends on other unregistered packages:
```

Comment:
is there a reason for removing this segment? OpenMendel says that we need all three packages installed in that order.

Reply:
Sorry, that was a mistake!